src/modules/order/order.controller.js

```
@@ -249,31 +451,150 @@ const completeOrder = async (req, res) => {
+      });
+    }
+    
+    logger.debug(`updateOrder: Calling order service to update order ${id}`);
     const order = await OrderService.updateOrder(id, {
       status,
       shippingAddress,
       paymentDetails,
     });
-    res.status(200).json(order);
+    
+    logger.info(`updateOrder: Order ${id} updated successfully${status ? ' with new status: ' + status : ''}`);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)